docs/source/MineYourMind/index.rst

```diff
@@ -42,27 +56,41 @@ Our Founders manage the finances and future direction of the network as a whole.
-the servers including heart issues, and questing problems. They are handed work from the Administrator Team. 
-They report directly to the Manager position.
+Our Senior Moderators are in charge of the day to day operation of the servers in game. Their main focus is on dealing with the community, handling escalated tickets from moderators, mentoring Jr. Moderators and general task lists from administrators and managers.
+
+They are also responsible for handling issues that don't require backend access, and they handle issues that are out of the hands of the Moderators and are not directly required to be fixed by an Administrator. 
+
+They also help fix or deal with bugs on the servers including heart issues and questing problems.
+
+They are handed work from the Administrator Team and they report directly to the Manager position.
 
 Moderator
 *********
-Our Moderators are the most important position within MyM. They are 100% our customer service focused group. They handle all day to day player interactions in-game, on the forum, and in tickets. Every interaction between staff and a player starts with a moderator. They are the MyM Player ambassadors and are expected to fight on behalf of the player until their problem is resolved. They also help identify server performance issues and pass along information to the Senior Moderator team for faster handling of server performance.
+Our Moderators are the most important position within MyM. They are 100% our customer service focused group. They handle all day to day player interactions in-game, on the forum, and in tickets.
+
+Every interaction between staff and a player starts with a moderator. They are the MyM Player ambassadors and are expected to fight on behalf of the player until their problem is resolved.
+
+They also help identify server performance issues and pass along information to the Senior Moderator team for faster handling of server performance.
+
 They report directly to the Manager position.
 
 Junior Moderator
 ****************
-Our Junior Moderators are a new position to MyM. This position serves as a trial period for new staff members before moving into the Moderator position. This gives the new staff member an idea of what is expected as far as work ethic and our staff members a chance to make sure the new staff member is a good fit for the team. They mainly focus on Moderating chat and shadowing the Moderator team members to learn the ropes.
+Our Junior Moderators are a new position to MyM. This position serves as a trial period for new staff members before moving into the Moderator position. This gives the new staff member an idea of what is expected as far as work ethic and our staff members a chance to make sure the new staff member is a good fit for the team. 
```

Comment:
I am wondering if we should remove the "new position to MyM" part. It isn't a new position anymore. I think it should be more along the lines of, "Our Junior Moderators are our newest staff members to MyM." I think this is a better explaination.